src/config/query-keys.js

```diff
@@ -1,3 +1,4 @@
 export const queryKeys = {
   popularMovies: "popular-movies",
+  topTvShow: "popular-tvshow",
```

Comment:
This should be plural -- `topTvShows: popular-tv-shows`.

The reason for this is that we might have conflicting keys for endpoints that have a single resource attached to them instead of an endpoint that returns multiple resources. This is how `react-query` works. It's hard to explain but we should follow this convention in order not to cause an error further down the line.